vscode/src/completions/get-inline-completions.ts

```diff
@@ -202,6 +202,14 @@ async function doGetInlineCompletions(params: InlineCompletionsParams): Promise<
         return null
     }
 
+    // Do not trigger when cusor is at the start of the file ending line, and the line above is empty
+    if (triggerKind !== TriggerKind.Manual && position.line && position.line === document.lineCount - 1) {
```

Comment:
```suggestion
    if (triggerKind !== TriggerKind.Manual && position.line !== 0 && position.line === document.lineCount - 1) {
```

Let's be explicit about `0` checks. The inexplicit `if (foo)` should only be used for null and undefined IMO.